### ISSUE TITLE
Optimize size of web packed modules

### ISSUE THREAD

**Issue by dselman** [accordproject/cicero#337]
![image (19)](https://user-images.githubusercontent.com/623108/55560792-e374e100-56e8-11e9-8cf3-50c927f7305b.png)

**Comment by dselman** [comment]
`ietf-language-tag-regex` should be quite easy to eliminate I think.

The Ergo compiler is over 1 MB. We should think about splitting the parser (used in hub) from the Ergo compiler.

**Comment by jeromesimeon** [comment]
I agree. We should provide different packages or Web bundles for parsing and compiling.

**Comment by jeromesimeon** [comment]
The new distinction between source and compiled templates should help very much. The compiler should go from a source template to a compiled template. The compiled template can be handed over to the engine. Either source or compiled template can be used for parsing. The step in between (compilation) can be isolated.
